src/arch-syscall-dump.c

```diff
@@ -127,6 +128,9 @@ int main(int argc, char *argv[])
 		case SCMP_ARCH_PPC64LE:
 			sys = ppc64_syscall_iterate(iter);
 			break;
+		case SCMP_ARCH_RISCV64:
+			sys = riscv64_syscall_iterate(iter);
+			break;
```

Comment:
All the changes in `arch-syscall-dump.c` probably should be split out into a separate patch.  Yes, it is a bug fix, but it is a *different* bug that you are fixing.

Reply:
Good plan.  It looks like we missed RISCV support in a few of the tools.  I'll make a separate pull request for them soon

Reply:
Addressed in pull request #212